Demo/Application/Base/AppDelegate.swift

```diff
@@ -16,7 +16,11 @@ import BraintreeCore
         
         return true
     }
-    
+
+    func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
```

Comment:
Why is this necessary? Running it on my simulator it never gets triggered

Reply:
Ah, good catch. We need to use the scene delegate `continue` instead ([indicated here](https://developer.apple.com/documentation/xcode/supporting-universal-links-in-your-app)). We won't hit `openURLContexts` ever with our current setup since out app is currently always running running with hitting the URL directly from our demo app. Updated here: https://github.com/braintree/braintree_ios/pull/1205/commits/59f76e05ad9c8cf4f401010451d01aa5c8fe8ea4

Reply:
Gotcha! So a merchant will need to provide us a hook from [scene(_:continue:)](https://developer.apple.com/documentation/uikit/uiscenedelegate/3238056-scene)
 if using SceneDelegates _or_ [application(_:continue:restorationHandler:)](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623072-application) if using AppDelegate only.
 
We'll also need to find out the SwiftUI equivalent (similar to our SwiftUI entry [here](https://developer.paypal.com/braintree/docs/guides/venmo/client-side/ios/v6#handle-app-context-switching)). We could 🏈 punt on the SwiftUI one though, if we want to make a backlog ticket?

Reply:
Yep, exactly! I think we can punt on the Swift UI portion for now until we have the implementation hammered out a bit more for the feature.

Reply:
I added this to our docs ticket (DTBTSDK-3609) to ensure we add these methods for both UIKit and SwiftUI. If you think this needs a separate ticket happy to break it out but made sense as an addition to our docs story.